ui-spacetimechart/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export type SpaceTimeRegion = {
+  type?: string;
+  timeStart: Date; // timestamp
+  timeEnd: Date; // timestamp
+  spaceStart: number; // mm
+  spaceEnd: number; // mm
+  image?: string;
```

Comment:
- should not be possibly undefined
- this image is really the image itself ? Or the url of the image ?
```suggestion
  image: string;
```

Reply:
it’s the URL. renamed to imageUrl, and made it not optional